nlp/front/storage-mongo/src/main/kotlin/FaqDefinitionMongoDAO.kt

```diff
@@ -486,6 +486,10 @@ object FaqDefinitionMongoDAO : FaqDefinitionDAO {
                                 ne from listOf(
                                     ClassifiedSentence::status,
                                     ClassifiedSentenceStatus.deleted
+                                ),
+                                ne from listOf(
+                                    ClassifiedSentence::status,
+                                    ClassifiedSentenceStatus.inbox
```

Comment:
Why not explicitly getting validated and model ? I think it would be easier for someone who reads the code.

And it will be more stable if we introduce new status in the futur (archived ...)